File: src/dotnet/CSharp.SelfSignJwtTracking/Tracking.cs
namespace CSharp.SelfSignJwtTracking;

public class Tracking
{
    public Tracking(string trackingId)
    {
        this.trackingId = trackingId;
    }

    // public Tracking()
    // {
    //     this.trackingId = null;
    // }
    public string trackingId { get; set;}
}

File: src/ruby/ruby-tokengenerator/tracking.rb
# frozen_string_literal: true

class Tracking

  def initialize(trackingId)
    @trackingId = trackingId
  end

  def to_hash
    hash = {}
    instance_variables.each { |var| hash[var.to_s.delete('@')] = instance_variable_get(var) }
    hash
  end
end
